### DISCUSSION TITLE
Can there be a way to define custom "default browser" only for PWA?

### DISCUSSION THREAD

**Discussion by markcellus** [filips123/PWAsForFirefox#590] [Ideas]
Often times I'd want the links on the pages in a PWA to open in another browser when I click them. We have the option to open the links in the system's "default browser". 

But it would be nice if there was a way to set a custom browser that all links in the PWA should open with. This would allow me to still keep my system default browser as-is, but have all links in a PWA open in a different browser at another path. 

e.g. `PWA_BROWSER=/usr/bin/some-other-browser`.

**Comment by filips123** [comment]
You can open settings in the PWA (press `F6` then enter `about:preferences`). You should then be able to set another browser as a HTTP(S) handler under Applications in the General tab:

![screenshot](https://github.com/user-attachments/assets/136aa974-603c-46fc-b932-56ba51d8b29a)

**Reply by markcellus** [reply]
Oh great. But I just checked and don't see any `https` protocol option or the option to add a new one. How did you get it to show up?

![image](https://github.com/user-attachments/assets/991adb6c-7d5c-42ad-899e-aa3114aa4e0b)

**Reply by markcellus** [reply]
Oops, nevermind. I had to check `Open out-of-scope URLs in a default browser (can break some web apps)`, click a link in the PWA, and then the `https` showed up. Thank you so much for the great extension and for your help!
